### PR TITLE
Add support for Python 3.4-3.6, drop 2.6 and 3.3

### LINKED ISSUE #1: Support current Python versions
* Drop support for Python 2.6 and 3.3
* Add support for Python 3.4, 3.5, 3.6

### PR DESCRIPTION
**Pull request by hugovk**
Fixes #1.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,7 +5,9 @@ CHANGES
 3.0.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added support for Python 3.4, 3.5 and 3.6.
+
+- Dropped support for Python 2.6 and 3.3.
```
Comment (icemac) [inline CHANGES.txt:10]: As this change is backwards incompatible the version number should be increased to `4.0` (here and in `setup.py`.)

File: setup.py [hunk 2]
```diff
@@ -61,6 +62,7 @@ def read(*rnames):
     extras_require=dict(
         test=['z3c.form [test]'],
         ),
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```
Comment (icemac) [inline setup.py:65]: I like it!

**Review by icemac — APPROVED** [review]
Thank you for your PR.
Looks good to me with one additional change request.

File: CHANGES.txt (hunk 1 above)
Comment (hugovk) [reply CHANGES.txt:10]: Done!

**Comment by icemac** [comment]
Thank you very much!

**Pull request merged**